tools/run_tests/xds_k8s_test_driver/framework/xds_k8s_testcase.py

```diff
@@ -826,12 +826,17 @@ def _start_test_client(
         self,
         server_target: str,
         *,
+        wait_for_active_ads_timeout: Optional[_timedelta] = None,
         wait_for_active_channel_timeout: Optional[_timedelta] = None,
         **kwargs,
     ) -> XdsTestClient:
         test_client = self.client_runner.run(
             server_target=server_target, **kwargs
         )
+        test_client.wait_for_active_ads_channel(
```

Comment:
Since we technically don't check for the ADS specifically, we probably should say `xds` instead of `ads`.

Also, looks like we'd need to rename `wait_for_active_server_channel`. Both say `wait_for_active_`, but we check for completely different things. Maybe let's rename `wait_for_active_server_channel` to `wait_for_server_channel_ready` and rename `wait_for_active_ads_channel` to `wait_for_xds_channel_active`?

Reply:
`wait_for_server_channel_ready` does seems more correct, will change both names.